decisions/Gondor.txt

```diff
@@ -1812,12 +1812,22 @@ political_decisions = {
 		}
 		
 		effect = {
-			prestige = 30
-			badboy = 10
-			
 			random_owned = {
-				limit = { owner = { UMB = { exists = no } } }
+				limit = {
+					owner = {
+						NOT = {
+							UMB_256 = { owned_by = UMB }
+							UMB_260 = { owned_by = UMB }
+							UMB_263 = { owned_by = UMB }
+							UMB_270 = { owned_by = UMB }
+							UMB_273 = { owned_by = UMB }
+							HAR_277 = { owned_by = UMB }
+						}
+					}
+				}
```

Comment:
Probably should also check if it's owned by GON -- I don't think the intention is that they get cores on the provinces that they already own, just provinces owned by other realms

Reply:
They won't get cores on province they already otherwise because of the conditions in the Claim Umbar potential